osu.Server.Queues.BeatmapProcessor/BeatmapProcessor.cs

```diff
@@ -34,7 +34,7 @@ protected override void ProcessResult(BeatmapItem item)
                     // ensure the correct online id is set
                     working.BeatmapInfo.OnlineID = (int)beatmapId;
 
-                    calculator.ProcessBeatmap(working);
+                    calculator.ProcessDifficulty(working);
```

Comment:
While I don't have much knowledge of what this processor is doing exactly, I'm guessing it's intended to process beatmaps which were newly submitted to web, and if so, I'm not sure difficulty is sufficient? Like, what is going to be computing and populating the scoring attributes automatically for new beatmaps after running the manual command to migrate existing beatmaps to the new structure?

Basically, naively, I'd expect this to be

```suggestion
                    calculator.ProcessAll(working);
```